pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -198,6 +200,17 @@ func (r *ReconcileBareMetalHost) Reconcile(request reconcile.Request) (result re
 		}
 	}
 
+	// Check if Status is empty and status annotation is present
+	// Manually restore data.
+	if !r.hostHasStatus(host) {
+		obj := r.getHostFromAnnotation(host)
+		if obj != nil {
+			host.Status = obj.Status
+		} else {
+			reqLogger.Info("Failed to find status annotation. Couldn't store status data manually")
```

Comment:
manually ? what does this mean?

Reply:
I have rephrased it a bit. Since we are putting the status back manually not the reconciler thats why I am using 'manually'